CHANGELOG.md

```diff
@@ -24,77 +24,98 @@ Packages are formatted by release version. You can see all the changes done to p
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+The changelog is generated during the release process using [ChangelogLinker](https://github.com/symplify/changeloglinker) since `7.0.0-alpha6` release.
+
+<!-- changelog-linker -->
+
+<!-- dumped content start -->
+
+## Unreleased
```

Comment:
Section Unreleased should be removed. Or not?

Reply:
this is a generated content. At the time of generation it is really "Unreleased" but in our release process checklist, there is an information that you should manually change it to the number of the version you are releasing